etl/config.py

```diff
@@ -100,6 +100,12 @@ def variable_metadata_url(variable_id):
 # number of workers for grapher inserts
 GRAPHER_INSERT_WORKERS = int(env.get("GRAPHER_WORKERS", 40))
 
+# number of workers for checking ditry steps
+DIRTY_STEPS_WORKERS = int(env.get("DIRTY_STEPS_WORKERS", 5))
+
+# number of processes for running steps
+RUN_STEPS_WORKERS = int(env.get("RUN_STEPS_WORKERS", 1))
```

Comment:
I'm a bit confused: You define here some default values of workers (which by the way would be good to justify, why 1, 5 or 40), but then in other places the default values of functions can be either a hardcoded number, or taken from config, or cpu_counts(). I don't know if all those choices were intentional, but it would be good to explain them.

Reply:
I've added some explanations. Values in `config.py` are default ones which can be overridden by `--workers`. The other ones for threads have good defaults and shouldn't be changed unless you are debugging core ETL. Ideally, we'd have just a single variable for `--workers` and the rest would be handled async instead of threads.